src/styles/blocks/header.scss

```diff
@@ -0,0 +1,59 @@
+@import "../utils/variables";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: $header-padding;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__image {
+    margin-top: $image-margin;
+    width: $image-size;
+    height: $image-size;
+  }
+}
+
+.nav {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 9th point